test/models/scholarship_application_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class ScholarshipApplicationTest < ActiveSupport::TestCase
```

Comment:
This is user preference, whether or not to keep these two autogenerated `*_test.rb` files or not.  

I typically delete them, until I have tests in them.  Some prefer to keep them as placeholders.